src/libstd/sys/windows/handle.rs

```diff
@@ -160,11 +160,6 @@ impl RawHandle {
         }
     }
 
-    pub fn read_to_end(&self, buf: &mut Vec<u8>) -> io::Result<usize> {
-        let mut me = self;
-        (&mut me).read_to_end(buf)
-    }
-
```

Comment:
This is an unused function newly detected due to improved reachability analysis (which is based on privacy data).